### DISCUSSION TITLE
Field description  for inputs and outputs

### DISCUSSION THREAD

**Discussion by BMWK1** [ptvoinfo/zigbee-configurable-firmware#297] [Ideas]
As i am building a more complex project, i notice that remembering the inputs and outputs use is a little tricky.
Adding a text description for each inputs and outputs could help to see easily what use is each pin, 15 or 20 characters are enough

**Comment by ptvoinfo** [comment]
@BMWK1 Sorry, about the delay with my answer. Where would you like to see this description? In the firmware configurator? Z2M or ZHA?

**Reply by BMWK1** [reply]
hi, many thanks for you answer, i suppose your are quite busy offering us such a great piece of soft
I was thinking about an extra field at the end of each input/output line in the settings tab.
I am using Z2M, so i don't know for ZHA

**Comment by ptvoinfo** [comment]
Ok. So, you'll use it just like a note for your configuration. 
Is it possible, one "Notes" field is enough? For example, I can add the "Notes" tab where you can place a description of your device in any format?

**Comment by BMWK1** [comment]
The possibility of a short use description of the input/output on each line would be more handy.
Wherever it is set, the purpose is to match an input/output with its description

**Comment by ptvoinfo** [comment]
The configuration interface is already overloaded with edit fields and options. Adding one more field, increases the complexity of the user interface.
